### PR TITLE
Add a redirect macro and initial stub pages

### PR DESCRIPTION
**Pull request by lifehackerhansol**
**Description**

We have a lot of pages that have been refactored over the past year, with major exploits being deprecated and easier(?) methods coming around.

A lot of once-popular pages on our guide is causing 404s as a result of our guide refactoring many pages that used to be there. Mainly an issue for older pages. Let's try to fix it.

Unfortunately, GitHub does not expose the option to use HTTP 301/2/7/8, and we can't really migrate to another provider at this time, so this hack will suffice.

The include macro should make it very simple to add redirects.

### CODE REVIEW

**Comment by ihaveamac** [comment]
All the pages (files in _pages/en_US) that were once in the guide and don't exist anymore as of current master: https://gist.github.com/ihaveamac/c9a9c214bf7f36499b0bf550e9122bbf

Pages I think should also get stubs:
* seedminer-(alternate).txt
* seedminer-(mii).txt
* seedminer-(twn).txt
* bannerbomb3.txt
* bannerbomb3-(legacy).txt
* installing-boot9strap-(kartdlphax).txt
* installing-boot9strap-(kartminer7).txt

File: _pages/en_US/410.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Page no longer exists" #
+sitemap: false
+---
+
+The page you are trying to view used to exist, but it has since been removed because it has been superseded by better exploits. Maybe the instructions you are following are outdated? (If you got here from a link on this guide, [let us know](https://github.com/hacks-guide/Guide_3DS/issues))
```
Comment (lilyuwuu) [inline _pages/en_US/410.txt:6]: Might want to reword this warning a bit? Idk

**Review by lilyuwuu — APPROVED** [review]
Lgtm

File: _pages/en_US/410.txt (hunk 1 above)
Comment (lifehackerhansol) [reply _pages/en_US/410.txt:6]: You wrote this and pushed it to master lol

Idk, the wording seems solid

**Pull request merged**